samples/mqtt5/mqtt5_pubsub/README.md

```diff
@@ -79,6 +79,9 @@ This section shows samples for all of the authentication possibilities.
 * [MQTT over Websockets with Sigv4 authentication](#mqtt-over-websockets-with-sigv4-authentication)
 * [Direct MQTT with Custom Authentication](#direct-mqtt-with-custom-authentication)
 * [MQTT over Websockets with Cognito](#mqtt-over-websockets-with-cognito)
+* [MQTT over Websockets with Sigv4 authentication](#mqtt-over-websockets-with-sigv4-authentication)
+* [Direct MQTT with Custom Authentication](#direct-mqtt-with-custom-authentication)
+* [MQTT over Websockets with Cognito](#mqtt-over-websockets-with-cognito)
```

Comment:
These 3 sections seems duplicate. Meanwhile we are missing the pkcs11, pkcs12 and windows certificate store section.